galaxy_ng/app/migrations/0025_add_content_guard_to_distributions.py

```diff
@@ -0,0 +1,31 @@
+from django.db import migrations
+
+def add_content_guard(apps, schema_editor):
+    """
+    Enforce that all distributions have a content guard applied
+    """
+    AnsibleDistribution = apps.get_model('ansible', 'AnsibleDistribution')
+    ContentGuard = apps.get_model('galaxy', 'CollectionDownloadContentGuard')
+
+    AnsibleDistribution.objects.filter(
+        content_guard=None,
+        pulp_type='ansible.ansible'
+    ).update(
+        content_guard=ContentGuard.objects.get_or_create(
+            pulp_type='ansible.ansible'
+        )[0]
```

Comment:
Is '[0]' required here? I may be completely wrong, but I think it implies that "ContentGuard.objects.get_or_create()" returns a queryset when in fact "objects.get()" and "objects.get_or_create()" return a single object.

Reply:
`get_or_create` returns a tuple @chouseknecht 

commonly we use like:
```py
content_guard, created = ContentGuard.objects.get_or_create(...)
```
content_guard: ContentGuard
created: bool # indicate whether the object was get or created 

It is not wrong to access the `[0]` index but to be more clear I recommend

```py
content_guard, _ = ContentGuard.objects.get_or_create(pulp_type='ansible.ansible')
...
.update(content_guard=content_guard)
```

Reply:
Much clearer, I was not aware of that pattern.  Update coming.